app/modules/sightings/models.py

```diff
@@ -894,44 +964,47 @@ def _parse_id_response(self, job_id_str, data):
                 {str(target_annot.guid): target_annot_data['score']}
             )
 
-        return result
+        return status, result
 
     def identified(self, job_id, data):
-        if self.stage != SightingStage.identification:
-            raise HoustonException(log, f'Sighting {self.guid} is not detecting')
         job_id_str = str(job_id)
         if job_id_str not in self.jobs:
             raise HoustonException(log, f'job_id {job_id_str} not found')
-
-        result = self._parse_id_response(job_id_str, data)
-
-        from app.modules.ia_config_reader import IaConfig
-
-        ia_config_reader = IaConfig(current_app.config.get('CONFIG_MODEL'))
         job = self.jobs[job_id_str]
         algorithm = job['algorithm']
+        annot_guid = job['annotation']
+        debug_context = f'Sighting:{self.guid}, Annot:{annot_guid}, algorithm:{algorithm}'
+
+        status, result = self._parse_id_response(job_id_str, data)
+
+        description = ''
```

Comment:
refactored to handle the lack of descriotion(that's only used for trace) with a warning rather than an exception